server/server.js

```diff
@@ -1,6 +1,93 @@
-const app = require('./app');
+const express = require('express');
+const bodyParser = require('body-parser');
+const morgan = require('morgan');
+const cors = require('cors');
+const db = require('../database/index');
```

Comment:
You can just write `const db = require('../database');` and it will read the `index.js` automatically.